packages/job-queue-plugin/src/bullmq/bullmq-job-queue-strategy.ts

```diff
@@ -52,21 +52,25 @@ export class BullMQJobQueueStrategy implements InspectableJobQueueStrategy {
     private readonly CANCEL_JOB_CHANNEL = 'cancel-job';
     private readonly CANCELLED_JOB_LIST_NAME = 'vendure:cancelled-jobs';
 
-    async init(injector: Injector): Promise<void> {
+    private initOptions(injector: Injector): BullMQPluginOptions {
         const options = injector.get<BullMQPluginOptions>(BULLMQ_PLUGIN_OPTIONS);
-        this.options = {
-            ...options,
-            workerOptions: {
```

Comment:
Please spread the configured worker options in there with `...(options.workerOptions ?? {})` instead of assigning it the bottom.

Reply:
Great point, pls see updated version.

Reply:
@areg-gareginyan-im the default values of `removeOnComplete` and `removeOnFail` should be kept. If you are passing `workerOptions` now they would be lost.